### DISCUSSION TITLE
include export to CSV and HTML after scanning

### DISCUSSION THREAD

**Discussion by cheahengsoon** [aquasecurity/trivy#4698] [Ideas]
### Description

after scanning the result, we required to export the result of scanning
for example result after scanning:  trivy image python:3.4-alpine

During Audit, we require the file to prove it.

### Target

None

### Scanner

None

**Comment by knqyf263** [comment]
`-f json` exports the report in the JSON format. You can convert it into CSV.

**Reply by fatihtokus** [reply]
@cheahengsoon if you don't mind do you use a tool for auditing these results?
